bciers/libs/components/src/datagrid/cells/OperationsActionCell.tsx

```diff
@@ -2,18 +2,18 @@ import Link from "next/link";
 import { GridRenderCellParams } from "@mui/x-data-grid";
 import { OperationStatus } from "@bciers/utils/enums";
 
-const OperationsActionCell = () => {
+const OperationsActionCell = (isInternalUser: boolean) => {
   const renderCell = (params: GridRenderCellParams) => {
-    let actionText;
-    switch (params.row.status) {
-      case OperationStatus.NOT_STARTED:
-        actionText = "Start Registration";
-        break;
-      case OperationStatus.DRAFT:
-        actionText = "Continue";
-        break;
-      default:
-        actionText = "View Operation";
+    let actionText = "View Operation";
+    if (!isInternalUser) {
```

Comment:
Nit: I think it's safer to render things based on who the user is rather than who the user is not when reasonable to do so. We've had some visual bugs previously when rendering based on who the user isn't, because it isn't specific and can include edge cases.  
  
Though I think this looks safe and not saying to necessarily change it, just something to consider in the future.